solutionbox/code_free_ml/analyze_data.py

```diff
@@ -0,0 +1,823 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Fully agree, but I think we should keep it as one big file for now until we figure out how this will ship/package.

For example, if we want users to call this on the command line like a script, then I think one file is easier.